docs/contributing.md

```diff
@@ -1,25 +1,24 @@
-Contributing & Community Resources
-==================================
-
-Branch Strategy
----------------
-  * `master` is the latest release tag
-  * `stable` is the version under development
-  * Feature branches should be branched from / merged into `stable`
-
-Issue Tracking
---------------
-  * Submit issues in github issues. 
-  * If it's being tracked internally, you may see references to OPSINFRA and DEVOPS JIRAs
-
-Communities & Resources
------------------------
-  * [Splunk Enterprise Documentation](http://docs.splunk.com/Documentation/Splunk)
-  * [Splunk Answers (Q&A)](http://answers.splunk.com/)
-  * [Splunk IRC](http://www.splunk.com/view/SP-CAAACDF) #splunk on [EFNet](http://www.efnet.org/)
-  * [Splunk Public uCern](https://connect.ucern.com/community/udevelop/splunk)
-
-Docs Navigation
-===============
-* [Docs Readme](README.md)
-* [Repository Readme](../README.md)
+# Contributing & Community Resources
+
+## Branch Strategy
+
+- `master` is the latest release tag
+- `stable` is the version under development
+- Feature branches should be branched from / merged into `stable`
+
+## Issue Tracking
+
+- Submit issues in github issues.
+- If it's being tracked internally, you may see references to OPSINFRA and DEVOPS JIRAs
+
+## Communities & Resources
+
+- [Splunk Enterprise Documentation](http://docs.splunk.com/Documentation/Splunk)
+- [Splunk Answers (Q&A)](http://answers.splunk.com/)
+- [Splunk IRC](http://www.splunk.com/view/SP-CAAACDF) #splunk on [EFNet](http://www.efnet.org/)
```

Comment:
Nit as we're reformatting, we could add:
`- [Splunk Slack](https://splunk-usergroups.slack.com) Request an invite [here](https://splunk-usergroups.signup.team)`